files/de/web/css/vertical-align/index.html

```diff
@@ -61,7 +61,7 @@
 </pre>
 </div>
 
-<p>{{EmbedLiveSample("vertical-align-inline", 1200, 120, "", "", "example-outcome-frame")}}</p>
+<p>{{EmbedLiveSample("vertical-align-inline", '1200', 160)}}</p>
```

Comment:
The quotes were only necessary because of the % sign. For numeric values, it's better to avoid them:

```suggestion
<p>{{EmbedLiveSample("vertical-align-inline", 1200, 160)}}</p>
```